Update understand-payment-method-in-checkout-process.md (#1320)
spryker/spryker-docs: docs/pbc/all/payment-service-providers/unzer/howto-tips-use-cases/understand-payment-method-in-checkout-process.md

## Standard type credentials

Whenever a customer goes through the checkout process, Unzer is called to know all the payment methods currently enabled in your account. This list is compared against the active payment methods for this store. All payment methods at the intersection of the Unzer data and the Spryker data are displayed.

> 

Reviewer: For better understanding i would recommend to add an example for standard as well.

## Marketplace type credentials

> Whenever a customer goes through the checkout process, Unzer is called to know all the payment methods currently enabled in your account. We compare the list against the active payment methods of the store and all the payment methods enabled for the merchants (in Unzer) inside the cart.

Reviewer: ```suggestion
Whenever a customer goes through the checkout process, Unzer is called to know all the payment methods currently enabled in your account. We compare the list against the active payment methods of the store and all the payment methods enabled for the merchants (in Unzer) related to the added products inside the cart.
```

Author: This is implied because the products belong to the merchants

> Whenever a customer goes through the checkout process, Unzer is called to know all the payment methods currently enabled in your account. We compare the list against the active payment methods of the store and all the payment methods enabled for the merchants (in Unzer) inside the cart.

Reviewer 2: Avoid using first person - https://developers.google.com/style/person
```suggestion
Whenever a customer goes through the checkout process, Unzer is called to know all the payment methods currently enabled in your account. The list is compared against the active payment methods of the store, and all the payment are methods enabled for the merchants (in Unzer) inside the cart.
```

* PayPal
* Apple Pay

{% info_block infoBox %}

> Spryker uses the marketplace Unzer credentials for all the payment methods that are marketplace compatible. Spryker will use merchant-specific Unzer credentials for all the other payment methods.

Reviewer 2: Avoid future tenses - https://developers.google.com/style/future
```suggestion
 For all the marketplace compatible payment methods, Spryker uses the marketplace Unzer credentials. For all the other payment methods, Spryker uses merchant-specific Unzer credentials.
```

> Based on the example above, Spryker will use the marketplace Unzer credentials for the Credit Card and Sofort payment methods. Spryker will use the merchant-specific Unzer credentials for PayPal and Apple Pay.

Reviewer 2: ```suggestion
Based on the preceding example, Spryker uses the marketplace Unzer credentials for the Credit Card and Sofort payment methods. For PayPal and Apple Pay, Spryker uses the merchant-specific Unzer credentials 
```

#### Example

In your Unzer account, the following payment methods are enabled for the marketplace business model.

> * Credit Card
> * Unzer Bank Transfer

Reviewer: ```suggestion
* Marketplace Credit Card
* Marketplace Bank Transfer
```

Author: No @michbeck this is internal behavior of our integration we must not put those details in the doc, for the business Credit Card and Marketplace credit card are the same

* Credit Card
* Unzer Bank Transfer

{% info_block infoBox %}

> Spryker uses the marketplace Unzer credentials for all the payment methods that are marketplace compatible. Spryker will use merchant-specific Unzer credentials for all the other payment methods.

Reviewer 2: avoid future tenses
```suggestion
For all the marketplace compatible payment methods, Spryker uses the marketplace Unzer credentials . For all the other payment methods, Spryker uses merchant-specific Unzer credentials.
```

> Based on the example above, Spryker will use the marketplace Unzer credentials for the Credit Card and Unzer Bank Transfer payment methods.

Reviewer 2: ```suggestion
Based on the preceding example, Spryker uses the marketplace Unzer credentials for the Credit Card and Unzer Bank Transfer payment methods.
```
